Source/ACE/Entity/Landblock.cs

```diff
@@ -104,10 +104,13 @@ public Landblock(LandblockId id)
             var factoryObjects = WorldObjectFactory.CreateWorldObjects(objects);
             factoryObjects.ForEach(fo =>
             {
-                if (!worldObjects.ContainsKey(fo.Guid))
+                if (fo != null)
```

Comment:
I think its important to decide if a correct implementation of CreateWorldObjects can or cannot introduce null objects into its list.  As it returns a list, I would imagine cannot is the correct behavior. 

I am a strong opponent of error-oblivious computing (intentionally ignoring errors), it leads to buggy, inconsistent, and difficult to debug code.  Intentionally ignoring values that should be impossible from CreateWorldObjects is a form of error-oblivious computing.

With that said, I would highly recommend removing this check, either replace it with an assertion (Debug.Assert) if your worried about null values, or remove it entirely.

As written, this code indicates CreateWorldObjects can/should return null values in its list, and that's just confusing.